www/js/goals.js

```diff
@@ -869,9 +857,9 @@ angular.module('emission.main.goals',['emission.services', 'emission.plugin.logg
         handlePendingRefer();
     };
 
-    if (storage.get(HABITICA_REGISTERED_KEY) == true) {
-        getChallenges();
-    }
+    //if (storage.get(HABITICA_REGISTERED_KEY) == true) {
+       // getChallenges();
```

Comment:
Yes, I made this change because now the getChallenges is called in the getUserInfo to get challenges every time when the page is refreshed. getUserInfo only works if the user is registered into habitica.